dcos_launch/platforms/aws.py

```diff
@@ -164,6 +165,7 @@ def create_stack(
         Starts stack creation if validation is successful
         """
         log.info('Requesting AWS CloudFormation: {}'.format(name))
+        role_arn = os.getenv('DCOS_LAUNCH_ROLE_ARN', None)
```

Comment:
nit: getenv "None" doesn't need to be specified because getenv returns None as a default

Reply:
done